Overview.bs

```diff
@@ -1627,8 +1639,15 @@ The algorithm:
 5. For each [[open-type/otff#table-directory|table]] in <var>base font subset</var> which has a tag that was not found in any of
     the entries processed in step 4, add a copy of that table to <var>extended font subset</var>.
 
-6. Modify the mappings in the IFT and IFTX table(s) in <var>extended font subset</var> if present to remove any mapping entries
-    that map to this patch. <!-- TODO: this needs to be made more detailed once we have the mapping format sections -->
+6. Modify the mappings in the 'IFT ' or 'IFTX' table which has an [=Glyph keyed patch/id=] which matches <var>id</var> to remove any
+    entries that map to this patch. The entries to be removed can be determined by finding entries which have a URI that matches the
```

Comment:
I'm not sure I understand this part. We're dealing with glyph-keyed patches in this section, correct? It sounds to me like we're saying the client should match the URI for the patch with the URI in the patch map, but isn't one of those "filled in" and one of them still a template? And then I guess the idea would be to derive the ID for the patch by extracting it during the matching process? If that's what's intended, surely no implementation would do that in practice. It would either stash the URI template and the ID used to fill in the template somewhere while waiting for the file to download, or (more likely) just stash the identity of the patch map and the ID (skipping the process of matching the template).

It sounds like we might be adapting the procedures for the other patch types to our glyph-keyed patches a little too directly. 

Or maybe I don't understand what's going on in this section of the documentation.

Reply:
So at a high level what I'm trying to communicate here is that the client will need to go set the bit in appliedBitmaps (or the equivalent 'ignored' bit in format 2) for the patch which is being applied. I'll take a pass at rewriting this to be more clear.

One important thing to note is that client implementations can choose to do things differently so long as the end result is the same (see: https://w3c.github.io/IFT/Overview.html#w3c-conformant-algorithms).  So for example with format 1 they could use entry index to quickly locate the bit to set, and for format2 they would probably want to record the position of the flags byte w/ the entry to quickly get at it later. For the spec that means we don't have to specify the most efficient way to perform an algorithm, and should instead bias towards the most clear explanation of what should happen. So in this case matching the patch's URL to the URL in the patch map entry explains which entries to update without having to dive into the details of the two different formats.